release-calendar/src/types.ts

```diff
@@ -26,6 +26,20 @@ export enum Channel {
   CREATED = 'created',
 }
 
+interface Inames {
+  [key: string]: string;
+}
+
+export const channelNames: Inames = {
+  'lts': 'Long Term Stable',
```

Comment:
Use enum values instead

Reply:
When I try to make it the enum, it give me this error. I dont think I'm allowed to.
An index signature parameter type cannot be a union type. Consider using a mapped object type instead.ts